### ISSUE TITLE
Dist file `gettext_extract.js` contains an absolute import path

### ISSUE THREAD

**Issue by Chocobozzz** [jshmrtn/vue3-gettext#58]
Can be saw on https://www.npmjs.com/package/vue3-gettext?activeTab=code

`dist/bin/gettext_extract.js` contains:

```js
var tslib_es6_js = require('/home/leo/projects/vue3-gettext/node_modules/tslib/tslib.es6.js');
```

Which of course causes a bug :)

Version: v3.0.0-beta.5

**Comment by lzurbriggen** [comment]
what the hell

**Comment by lzurbriggen** [comment]
Attempted to fix this in [3.0.0-beta.6](https://www.npmjs.com/package/vue3-gettext/v/3.0.0-beta.6)
